source/layers/validation/common/ze_entry_points.h

```diff
@@ -423,6 +423,9 @@ class ZEValidationEntryPoints {
     virtual ze_result_t zeCommandListUpdateMutableCommandWaitEventsExpEpilogue( ze_command_list_handle_t hCommandList, uint64_t commandId, uint32_t numWaitEvents, ze_event_handle_t* phWaitEvents , ze_result_t result) {return ZE_RESULT_SUCCESS;}
     virtual ze_result_t zeCommandListUpdateMutableCommandKernelsExpPrologue( ze_command_list_handle_t hCommandList, uint32_t numKernels, uint64_t* pCommandId, ze_kernel_handle_t* phKernels ) {return ZE_RESULT_SUCCESS;}
     virtual ze_result_t zeCommandListUpdateMutableCommandKernelsExpEpilogue( ze_command_list_handle_t hCommandList, uint32_t numKernels, uint64_t* pCommandId, ze_kernel_handle_t* phKernels , ze_result_t result) {return ZE_RESULT_SUCCESS;}
-    virtual ~ZEValidationEntryPoints() {}
+     // Experimental Intel extension for counter-based events
```

Comment:
make these changes in https://github.com/oneapi-src/level-zero/blob/master/scripts/templates/validation/entry_points.h.mako